package/MDAnalysis/analysis/base.py

```diff
@@ -334,8 +334,15 @@ def _setup_frames(self, trajectory, start=None, stop=None, step=None,
         step : int, optional
             number of frames to skip between each analysed frame
         frames : array_like, optional
-            array of integers or booleans to slice trajectory
-
+            array of integers or booleans to slice trajectory;
+            cannot be combined with `start`, `stop`, `step`
+  
+        Raises
+        ------
+        ValueError
```

Comment:
I didn't put the full ValueError into `run` because it seemed too much information, just in docs